rest/buildlogger_routes.go

```diff
@@ -29,24 +30,29 @@ type logGetByIDHandler struct {
 	tr        util.TimeRange
 	printTime bool
 	sc        data.Connector
+	userToken string
+	evgURL    string
 }
 
-func makeGetLogByID(sc data.Connector) gimlet.RouteHandler {
+func makeGetLogByID(sc data.Connector, evgURL string) gimlet.RouteHandler {
 	return &logGetByIDHandler{
-		sc: sc,
+		sc:     sc,
+		evgURL: evgURL,
 	}
 }
 
 // Factory returns a pointer to a new logGetByIDHandler.
 func (h *logGetByIDHandler) Factory() gimlet.RouteHandler {
 	return &logGetByIDHandler{
-		sc: h.sc,
+		sc:     h.sc,
+		evgURL: h.evgURL,
 	}
 }
 
 // Parse fetches the id and time range from the http request.
 func (h *logGetByIDHandler) Parse(_ context.Context, r *http.Request) error {
 	h.id = gimlet.GetVars(r)["id"]
+	h.userToken = r.Header.Get(cedar.AuthTokenCookie)
```

Comment:
I feel like this token isn't going to be the right token for evergreen? don't we need to see if we can get the mci auth token, not the cedar auth token?

Reply:
Maybe not... it was to my understanding that since we use LDAP & the evergreen group for authentication the auth cookie token would be the same.

Reply:
pretty sure that the token/cookie content is a random string generated in gimlet code during authentication... it wouldn't be the same between different applications.